src/com/tony/server/ResponseDeterminer.java

```diff
@@ -28,6 +29,8 @@ public Response determineResponse(Request request) {
             }
         } else if(uriList.contains(request.getURI())){
             return new MethodNotAllowedResponse();
+        } else if(request.getURI().startsWith("/parameters?")){
```

Comment:
Sorry, I meant to work on the parameter decode changes in this branch but I did it in the branch 1 ahead of this one. https://github.com/tbaik/java-server/commit/d07ddfc79a621f584cf14125a0d4127e1edcef64#diff-bf58bc1910ee85f9e4946f490ba5410aR40